build.xml

```diff
@@ -295,6 +313,44 @@
 
     </target>
 
+    <target name="lang.docinfo.texi">
+
+        <mkdir dir="${build.dir}/${lang}/info/"/>
+
+        <java classname="com.icl.saxon.StyleSheet" fork="true" dir="${basedir}" maxmemory="192m">
+            <classpath refid="lib.classpath"/>
+            <arg value="-o"/>
+            <arg value="${build.dir}/${lang}/info/ceylon-spec.texi"/>
+            <arg value="${basedir}/${lang}/master.xml"/>
+            <arg value="${basedir}/${lang}/styles/${db.style.info}"/>
+        </java>
+
+    </target>
+
+    <target name="lang.docinfo.make" if="hasInfo">
+
+        <exec executable="makeinfo" dir="${build.dir}/${lang}/info/">
+            <arg value="ceylon-spec.texi"/>
+        </exec>
+      
+    </target>
+
+    <target name="lang.docinfo"
+            depends="lang.docinfo.texi,lang.docinfo.make"
+            description="Generates the Texinfo documentation only for a language (set lang)"/>
+
+    <target name="lang.docinfo.install"
+            depends="lang.docinfo"
+            if="hasInfo">
+
+        <exec executable="sudo" dir="${build.dir}/${lang}/info/">
```

Comment:
Well, do you notice how it should be the user who explicitly runs `sudo` and not `make`?

Not only that, but I'd want to install to my user info path, not the system, like we do for the rest of the `publish` commands.